go/kbfs/libkbfs/settings_db.go

```diff
@@ -125,16 +127,27 @@ func (db *SettingsDB) Settings(ctx context.Context) (keybase1.FSSettings, error)
 	if uid == keybase1.UID("") {
 		return keybase1.FSSettings{}, errNoSession
 	}
+
 	var notificationThreshold int64
 	notificationThresholdBytes, err :=
 		db.Get(getSettingsDbKey(uid, spaceAvailableNotificationThresholdKey), nil)
 	if err == nil {
 		notificationThreshold, _ =
 			strconv.ParseInt(string(notificationThresholdBytes), 10, 64)
 	}
+
+	var sfmiBannerDismissed bool
+	sfmiBannerDismissedBytes, err :=
+		db.Get(getSettingsDbKey(uid, sfmiBannerDismissedKey), nil)
+	if err == nil {
+		sfmiBannerDismissed, _ =
```

Comment:
Why ignore this error?  Just because the code copied from above?  Seems like it should be returned in both places.

Reply:
I guess the idea is if some bad value got into the db somehow, this would give just override it with a good value. But I also can't thing of a legit reason that this could happen. So perhaps this error should just be returned.

Reply:
Should at least be logged if it can't be returned.  Or at an absolute bare minimum, a comment with your explanation.